docs/developer_workflow.md

```diff
@@ -4,89 +4,38 @@ This document outlines how to
 
 ## Build the source code
 
-1. Download source code:
+Download source code:
 
     ```shell
     git clone https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix.git
     cd cluster-api-provider-nutanix
     ```
 
-1. Build a container image with the source code:
+## Build source code
```

Comment:
Please add instructions to install devbox, direnv and container runtime.

Reply:
:heavy_check_mark: